cmake/platform_apple.cmake

```diff
@@ -64,17 +64,10 @@ function(setup_support_build)
 endfunction()
 
 function(setup_rest_build)
+    setup_rest_build_unix()
+    
     set_target_properties(
         LiteCoreREST PROPERTIES LINK_FLAGS
         "-exported_symbols_list ${CMAKE_CURRENT_SOURCE_DIR}/c4REST.exp"
     )
-
```

Comment:
assuming this was resolved only by importing to apple platform, but now seems to be imported to all platforms, and how did that resolved?

Reply:
This one is less relevant because it is not a product that we ship, it is only used interally.  The main target of this PR is LiteCore (not LiteCoreREST)